scripts/test/Muon/PeriodicTableWidget_test.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import, print_function
+
+import unittest
+
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table_view import PeriodicTableView
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table_presenter import PeriodicTablePresenter
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table_widget import PeriodicTable
+from Muon.GUI.Common import mock_widget
+
+from PyQt4 import QtGui
+
+from silx.gui.widgets import PeriodicTable as silxPT
+
+from silx.gui.widgets.PeriodicTable import ColoredPeriodicTableItem
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+
+class MockEvent(object):
+    def __init__(self):
+        self.call_count = 0
+
+    def handler(self):
+        self.call_count += 1
+
+
+class PeriodicTableWidgetTest(unittest.TestCase):
+    def setUp(self):
+        self._qapp = mock_widget.mockQapp()
+        self.widget = PeriodicTable()
+        self.view = self.widget.presenter.view
```

Comment:
I think you are better off explicitly setting the mock for this https://github.com/mantidproject/mantid/blob/master/scripts/test/Muon/transformWidget_test.py#L29

then you can set it https://github.com/mantidproject/mantid/blob/master/scripts/test/Muon/transformWidget_test.py#L37